src/github_profile.py

```diff
@@ -15,7 +15,7 @@ def generate_profile(theme, **kwargs):
             continue
 
         # Skip items that don't exist in theme
-        item_path = Path(f"src/themes/{theme}/{item}.txt")
+        item_path = Path(f"../src/themes/{theme}/{item}.txt")
```

Comment:
⚪ Isn't it better to run the App on top of src folder and delete the `../` part of URL?